### PR TITLE
develop to master

### PR DESCRIPTION
**Pull request by budgetpreneur**

### CODE REVIEW

File: src/Api/PubnubApi/EndPoint/Access/AuditOperation.cs [hunk 1]
```diff
@@ -80,6 +80,16 @@ public void Execute(PNCallback<PNAccessManagerAuditResult> callback)
 #endif
         }
 
+
+        public async Task<PNResult<PNAccessManagerAuditResult>> ExecuteAsync()
+        {
+#if NETFX_CORE || WINDOWS_UWP || UAP || NETSTANDARD10 || NETSTANDARD11 || NETSTANDARD12
+            return await AuditAccess(this.channelName, this.channelGroupName, this.authenticationKeys, this.queryParam).ConfigureAwait(false);
+#else
+            return await AuditAccess(this.channelName, this.channelGroupName, this.authenticationKeys, this.queryParam).ConfigureAwait(false);
```
Comment (davidnub) [inline src/Api/PubnubApi/EndPoint/Access/AuditOperation.cs:89]: This `else` statement looks the same as the `if` am I missing something here or are you? :-P

File: src/Api/PubnubApi/EndPoint/Access/GrantOperation.cs [hunk 2]
```diff
@@ -120,6 +120,19 @@ public void Execute(PNCallback<PNAccessManagerGrantResult> callback)
...
+        public async Task<PNResult<PNAccessManagerGrantResult>> ExecuteAsync()
+        {
+            if (config != null && pubnubLog != null)
+            {
+                LoggingMethod.WriteToLog(pubnubLog, string.Format("DateTime: {0}, WARNING: Grant() signature has changed! This specific call will be making a request to PAMv2. Please update your code if this is not the intended action.", DateTime.Now.ToString(CultureInfo.InvariantCulture)), config.LogVerbosity);
+            }
+#if NETFX_CORE || WINDOWS_UWP || UAP || NETSTANDARD10 || NETSTANDARD11 || NETSTANDARD12
+            return await GrantAccess(this.pubnubChannelNames, this.pubnubChannelGroupNames, this.pamAuthenticationKeys, this.grantRead, this.grantWrite, this.grantDelete, this.grantManage, this.grantTTL, this.queryParam).ConfigureAwait(false);
+#else
+            return await GrantAccess(this.pubnubChannelNames, this.pubnubChannelGroupNames, this.pamAuthenticationKeys, this.grantRead, this.grantWrite, this.grantDelete, this.grantManage, this.grantTTL, this.queryParam).ConfigureAwait(false);
```
Comment (davidnub) [inline src/Api/PubnubApi/EndPoint/Access/GrantOperation.cs:132]: This `else` also looks the same as the `if` maybe something in C# that i'm not understanding?

File: src/Api/PubnubApi/EndPoint/Access/AuditOperation.cs (hunk 1 above)
Comment (budgetpreneur) [reply src/Api/PubnubApi/EndPoint/Access/AuditOperation.cs:89]: will remove if else

File: src/Api/PubnubApi/EndPoint/Access/GrantOperation.cs (hunk 2 above)
Comment (budgetpreneur) [reply src/Api/PubnubApi/EndPoint/Access/GrantOperation.cs:132]: will remove if else

**Pull request merged**